calicoctl/commands/clientmgr/client.go

```diff
@@ -35,7 +36,7 @@ func NewClient(cf string) (*client.Client, error) {
 
 	c, err := client.New(*cfg)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("Syntax error in %s: %s", cf, err)
```

Comment:
I think the general recommendation is to start error strings with a lowercase character so that they are more easily wrapped in the calling code.

This can return `syntax error in calicoctl.cfg: file is not yaml`

And then in can be wrapped in the calling code:

`failed to load config: syntax error in calicoctl.cfg: file is not yaml` or something like that.